jersey-sample-poc/src/main/java/com/mh/jerseysamplepoc/RestClient.java

```diff
@@ -0,0 +1,81 @@
+package com.mh.jerseysamplepoc;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class RestClient {
+    private Client apiClient;
+    private WebTarget webTarget;
+    private Response response;
+    public static final String TYPE = "type";
+    public static final String JSON_RESPONSE = "json";
+    public static final String PROPERTIES_FILE = "app.properties";
+    public static Properties properties = new Properties();
+
+    private Properties readProperties() {

```

Comment:
either you can make this method static or use the return value directly from the caller. Also you are using a static member to keep the properties, in that case you don't have to return the result. If you are returning a value from a method you should use it.

Reply:
Made the webTarget and response variables local.